src/testing/testing_runtime.ts

```diff
@@ -21,7 +21,7 @@ import { DBOSScheduler } from "../scheduler/scheduler";
 /**
  * Create a testing runtime. Warn: this function will drop the existing system DB and create a clean new one. Don't run tests against your production database!
  */
-export async function createTestingRuntime(userClasses: object[], configFilePath: string = dbosConfigFilePath, dropSysDB: boolean = true): Promise<TestingRuntime> {
+export async function createTestingRuntime(userClasses?: object[], configFilePath: string = dbosConfigFilePath, dropSysDB: boolean = true): Promise<TestingRuntime> {
```

Comment:
Are these arguments staying in for backwards-compatibility reasons?  It seems we always pass `undefined` in. If so, can we use overloads to simplify things?

Reply:
Yes, I would keep the first argument the same so as to not break compatibility, it's just now optional.

Also, on the off-chance that something goes wrong, you can still provide a list.

You can't always use an overload to simplify things unless you can disambiguate the positional arguments at runtime, but you can now say () in most cases, which is a win.